alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultNotificationAccessor.java

```diff
@@ -29,6 +29,8 @@
 import java.util.Optional;
 import java.util.stream.Collectors;
 
+import javax.annotation.Nullable;
```

Comment:
Should we be using the jetbrains nullable?

Reply:
Good catch. This change was made before that decision.